jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/Jsonb1Annotator.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright © 2010-2020 Nokia
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jsonschema2pojo;
+
+import static org.apache.commons.lang3.StringUtils.isNotEmpty;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.sun.codemodel.JAnnotationArrayMember;
+import com.sun.codemodel.JDefinedClass;
+import com.sun.codemodel.JFieldVar;
+import com.sun.codemodel.JMethod;
+
+import org.jsonschema2pojo.rules.FormatRule;
+
+import javax.json.bind.annotation.JsonbDateFormat;
+import javax.json.bind.annotation.JsonbProperty;
+import javax.json.bind.annotation.JsonbPropertyOrder;
+import javax.json.bind.annotation.JsonbTransient;
+
+import java.util.Iterator;
+
+/**
+ * Annotates generated Java types using the Jackson 2.x mapping annotations.
```

Comment:
Looks like a copy/paste from Jackson annotator

Reply:
Yeah, I based my work on the Jackson annotator as the annotations and handling are quite similar, although not entirely the same. At the end, i copied JSON-B-annotator v1 for v2. This ways, I forgot to fix thecomments and then copied the errors from one version to the other, sorry. I guess the iterator could also be improved by using for-each.